### PR TITLE
[GulpV1] Migrated to Node10

### PR DESCRIPTION
**Pull request by sdobrodeev**
**Task name**: GulpV1

**Description**: Migrated GulpV1 to Node 10 execution handler.

**Documentation changes required:** No

**Added unit tests:** No

**Attached related issue:** No

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by anatolybolshakov — APPROVED** [review]

File: Tasks/GulpV1/Tests/L0.ts
Comment (anatolybolshakov) [inline Tasks/GulpV1/Tests/L0.ts:12]: Could you please create a ticket to cover this task by unit tests also?

**Review by DaniilShmelev — APPROVED** [review]
LGTM!

**Review by EzzhevNikita — APPROVED** [review]

**Comment by anatolybolshakov** [comment]
Let's merge this after merging of related canary PR

**Pull request merged**
